src/pages/resources/resources.ts

```diff
@@ -1,14 +1,15 @@
-import {derived, Readable} from 'svelte/store'
-import {API, Asset} from '@greymass/eosio'
-import {Resources, SampleUsage, PowerUpState, RAMState, REXState} from '@greymass/eosio-resources'
-import {activeBlockchain} from '~/store'
+import { derived, Readable } from 'svelte/store'
+import { Int64, API, Asset } from '@greymass/eosio'
+import { Resources, SampleUsage, PowerUpState, RAMState, REXState } from '@greymass/eosio-resources'
+import { activeBlockchain } from '~/store'
+import { BNPrecision } from '@greymass/eosio-resources'
 
-import {getClient} from '../../api-client'
-import {ChainConfig, ChainFeatures, resourceFeatures} from '~/config'
+import { getClient } from '../../api-client'
+import { ChainConfig, ChainFeatures, resourceFeatures } from '~/config'
 
 const getResourceClient = (chain: ChainConfig) => {
     const api = getClient(chain)
-    const options: any = {api}
+    const options: any = { api }
```

Comment:
I know that we don't want to spend too much time on this project, but we could probably use a proper type here instead of `any`.

Reply:
ResourcesOptions it not exported. Otherwise, need to change greymass/eosio-resources，Can this be ignored?